src/compiler/checker.ts

```diff
@@ -113,6 +116,8 @@ module ts {
         let globalRegExpType: ObjectType;
         let globalTemplateStringsArrayType: ObjectType;
         let globalESSymbolType: ObjectType;
+        let jsxElementType: ObjectType;
+        let jsxIntrinsicElementsType: ObjectType;

```

Comment:
They're not global; they're in the JSX namespace. `globalJsxElementType` though?